stations-along-route/map.js

```diff
@@ -0,0 +1,190 @@
+import mapboxgl from 'mapbox-gl';
+
+mapboxgl.accessToken = 'pk.eyJ1IjoiY2hhcmdldHJpcCIsImEiOiJjazhpaG8ydTIwNWNpM21ud29xeXc2amhlIn0.rGKgR3JfG9Z5dCWjUI_oGA';
+
+const map = new mapboxgl.Map({
+  container: 'map',
+  style: 'mapbox://styles/chargetrip/ck98fwwp159v71ip7xhs8bwts',
+  zoom: 7.5,
+  center: [6, 52.2288],
+});
+
+/**
+ * Draw route polyline and show charging stations on the map.
+ *
+ * @param coordinates {array} Array of coordinates
+ * @param legs {array} route legs (stops) - each leg represents either a charging station, or via point or final point
+ */
+export const drawRoute = (coordinates, legs, alternatives) => {
+  const slider = document.getElementById('alternative');
+  const sliderSwitch = document.getElementById('switch');
+
+  if (map.loaded()) {
+    drawPolyline(coordinates);
+    showLegs(legs);
+  } else {
+    map.on('load', () => {
+      drawPolyline(coordinates);
+      showLegs(legs);
+    });
+  }
+  slider.addEventListener('input', () => {
```

Comment:
To me using the word slider is a bit confusing. The re-route example has a slider and this example has a switch.

Reply:
It was because switch is alreadyy a thing. I now changed it to switchOn